submissions/DmytryjK/task_js_dom/js/script.js

```diff
@@ -0,0 +1,169 @@
+        imgAlt : "etretat normandy france",
+        firstParagraph : "Étretat (Етрета) - невеликий курорт з прекрасними гальковими пляжами на узбережжі Алебастрового Ла Манша, одна з головних визначних пам'яток Нормандії, на півночі Франції. Він розташований біля самого пляжу, де води Англійського каналу сформували дивовижну красу прямовисні скелі з природними арками — один із найкрасивіших краєвидів Франції.",
+        secondParagraph : ""
+    },
+    {
+        h2 : "Гренландія",
+        subtitle : "Льодяний каньйон",
+        imgSrc : "images/Grenland canyon.jpg",
+        imgAlt : "Grenland canyon",
+        firstParagraph : "Цей воістину дивовижний острів притягує мандрівників з усього світу, як магніт.  Аналогія банальна, але краще вигадати складно. Значна частина Гренландії справді схована під надійним щитом із льоду, який останнім часом починає активно танути. Це результат глобального потепління, на яке вже багато років намагаються звернути увагу багато громадських організацій.",
+        secondParagraph : "У льодовиках утворюються численні каньйони - їх краса просто приголомшує. Таке враження, що природа, як найталановитіший художник на цій планеті, малює своїм пензлем хитромудрі, часом навіть сюрреалістичні картини."
+    },
+    {
+        h2 : "Норвегія",
+        subtitle : "Місто - Олесунн",
+        imgSrc : "images/Alesund norway.jpg",
+        imgAlt : "Alesund norway",
+        firstParagraph : "О́лесунн (норв. Ålesund) — місто і порт на заході Норвегії. Розташоване на північ від гирла фіорду Стур. Місто розкинулося на декількох островах, а саме — Норвьойя, Аспьойя, Гейсса (Гесса) та Окснойя, які є з'єднані мостами. За легендою, поселення бере свій початок у 9 столітті коли Ролло (Рольф) заснував поблизу маєток, де мешкав війт. Статус міста Олесунн отримав в 1848 р. Після того як місто зазнало пожежі в 1904 р., воно було відбудоване у оригінальному стилі арт-нуво.",
+        secondParagraph : "Олесунн — регіональний торговий центр і туристична база для маршрутів в регіонах Суннмьоре, долині Норанґ, льодовиках Ойє та островах Рунде та Ґіске. В місті знаходиться найбільша риболовна гавань Норвегії, пристановище для риболовних трейлерів що виловлюють тріску та палтус з виду камбалових. Разом з Тромсьо, Олесунн є осередком ловлі арктичних тюленів."
+    }
+];
+
+document.addEventListener('DOMContentLoaded', startEventsForPage());
+function startEventsForPage() {
+    //Tabs
+    const mainContentBlock = document.querySelectorAll('.main__content-block');
+    const navigationButton = document.querySelectorAll('.main__navigation-button');
+
+    createElements();
+
+    addClass(mainContentBlock, 'main__content-active');
+
+    currentTabButton('main__navigation-button_active');
+
+    function currentTabButton (tabSelector) {
+        addClass(navigationButton, tabSelector);
+        navigationButton.forEach((button, index) => {
+            button.addEventListener('click', e => {
```

Comment:
You can try to add listener only for menu wrap

Reply:
fixed